packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -161,7 +161,11 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs,
       });
     }
 
-    // Prepare context
+    // Prepare and check context
+    if (context[KeysInitSparql.extensionFunctionCreator] && context[KeysInitSparql.extensionFunctionMap]) {
+      throw new Error('It is not allowed to provide both an extensionFunctionCreator an a extensionFunctionMap ' +
```

Comment:
I would suggest moving this check into ActorQueryOperation.ts.
That is because other actors could modify the context during the pipeline, and possibly introduce conflicting entries afterwards.

Reply:
Also, I would suggest an error message like 'Illegal extensionFunctionCreator and extensionFunctionMap context entries, only one of them is allowed'.

Reply:
You're right, hadn't thought about that.